content/how-to/account/examples/streaming.js

```diff
@@ -0,0 +1,29 @@
+const Alpaca = require('@alpacahq/alpaca-trade-api')
+const alpaca = new Alpaca()
+
+// Prepare the websocket connection and subscribe to account_updates.
+let websocket = alpaca.websocket
+client.onConnect(function () {
```

Comment:
`client` will be undefined here. You probably mean `websocket`.

For less confusion I would even use `alpaca.websocket.onConnect` and similar for the other stream handlers.